web_src/js/components/DashboardRepoList.js

```diff
@@ -0,0 +1,370 @@
+import Vue from 'vue';
+import {initVueSvg, vueDelimiters} from './VueComponentLoader.js';
+
+const {AppSubUrl, AssetUrlPrefix, PageData} = window.config;
+
+function initVueComponents() {
+  Vue.component('repo-search', {
+    delimiters: vueDelimiters,
+    props: {
+      searchLimit: {
+        type: Number,
+        default: 10
+      },
+      subUrl: {
+        type: String,
+        required: true
+      },
+      uid: {
+        type: Number,
+        default: 0
+      },
+      teamId: {
+        type: Number,
+        required: false,
+        default: 0
+      },
+      organizations: {
+        type: Array,
+        default: () => [],
```

Comment:
```suggestion
        default: [],
```
Or why would you use an additional function call?

Reply:
I just copied(moved) old code here, I won't make big change for the first commit, otherwise it will be difficult for us to view commit history and diff. Any refactor should happen in future after the move.